.gitignore

```diff
@@ -11,4 +11,7 @@ wp-content/cache/
 wp-content/upgrade/
 wp-content/uploads/
 wp-content/wp-cache-config.php
-bower_components/
\ No newline at end of file
+bower_components/
+Gramophone.sublime-project
+Gramophone.sublime-workspace
+Thumbs.db
```

Comment:
I agree with @RickBakkr. Please remove these - they aren't needed.

Reply:
I removed the IDE-specific entries from .gitignore locally and synced it with the github repo. It created new pull requests instead of making changes in the current one. Now we have 3 pull requests.